Kernel/OrbitalFrequencies.m

```diff
@@ -127,44 +127,133 @@
+                 a L) rm \[Minus] 2 M a^2 En)/(r3 \[Minus] 
+              rm) (Kkr \[Minus] (r2 \[Minus] r3)/(r2 \[Minus] rm) Pihm)
+         ));
+  \[CapitalUpsilon]\[Phi] = L (Piz1/Kk\[Theta]) +
+    a/(Kkr (rp \[Minus] rm))
+     ((2 M En rp \[Minus] a L)/(r3 \[Minus] 
+           rp) (Kkr \[Minus] (r2 \[Minus] r3)/(r2 \[Minus] 
+              rp) Pihp) \[Minus] (2 M En rm \[Minus] 
+           a L)/(r3 \[Minus] 
+           rm) (Kkr \[Minus] (r2 \[Minus] r3)/(r2 \[Minus] 
+              rm) Pihm));
+  (*];*)
+  (*If[a==M,
+  \[CapitalGamma]=;\[CapitalUpsilon]\[CurlyPhi]=;(21)whereh\
+\[PlusMinus]=];(21) is not valid for the case|a|=
+  Msince there exists divergent terms in1/(r+\[Minus]r\[Minus])
+  ]*)
+  \[CapitalOmega]r = \[CapitalUpsilon]r/\[CapitalGamma]; \
+\[CapitalOmega]\[Theta] = \[CapitalUpsilon]\[Theta]/\[CapitalGamma]; \
+\[CapitalOmega]\[Phi] = \[CapitalUpsilon]\[Phi]/\[CapitalGamma];
+  (*\[Epsilon]0=a^2 (1-En^2)/L^2;
+  zm=1-x^2;
+  a2zp=(L^2+a^2 (-1+En^2) (-1+zm))/((-1+En^2) (-1+zm));
+  \[Epsilon]0zp=-((L^2+a^2 (-1+En^2) (-1+zm))/(L^2 (-1+zm)));
+  (*zmOverZp=If[a==0,0,zm/((L^2+a^2 (-1+En^2) (-1+zm))/(a^2 (-1+
+  En^2) (-1+zm)))];*)zmOverZp=zm/((L^2+a^2 (-1+En^2) (-1+
+  zm))/(a^2 (-1+En^2) (-1+zm)));
+  kr=Sqrt[(r1-r2)/(r1-r3) (r3-r4)/(r2-r4)];(*Eq.(13)*)k\[Theta]=Sqrt[
+  zmOverZp];(*Eq.(13)*)\[CapitalUpsilon]r=(Pi Sqrt[(1-En^2)(r1-r3)(r2-
+  r4)])/(2EllipticK[
+  kr^2]);(*Eq.(15)*)\[CapitalUpsilon]\[Theta]=(Pi L \
+Sqrt[\[Epsilon]0zp])/(2EllipticK[k\[Theta]^2]);(*Eq.(15)*)rp=M+Sqrt[
+  M^2-a^2];
+  rm=M-Sqrt[M^2-a^2];
+  hr=(r1-r2)/(r1-r3);
+  hp=((r1-r2)(r3-rp))/((r1-r3)(r2-rp));
+  hm=((r1-r2)(r3-rm))/((r1-r3)(r2-rm));
+  (*Eq.(21)*)\[CapitalUpsilon]\[Phi]=(2\[CapitalUpsilon]\[Theta])/(Pi \
+Sqrt[\[Epsilon]0zp]) EllipticPi[zm,
+  k\[Theta]^2]+(2a \[CapitalUpsilon]r)/(Pi(rp-rm)Sqrt[(1-En^2)(r1-
+  r3)(r2-r4)])((2M En rp-a L)/(r3-rp) (EllipticK[kr^2]-(r2-r3)/(r2-
+  rp) EllipticPi[hp,kr^2])-(2M En rm-a L)/(r3-rm) (EllipticK[
+  kr^2]-(r2-r3)/(r2-rm) EllipticPi[hm,kr^2]));
+  \[CapitalGamma]=
+  4M^2 En+(2a2zp En \[CapitalUpsilon]\[Theta])/(Pi L \
+Sqrt[\[Epsilon]0zp]) (EllipticK[k\[Theta]^2]-EllipticE[
+  k\[Theta]^2])+(2\[CapitalUpsilon]r)/(Pi Sqrt[(1-En^2)(r1-r3)(r2-
+  r4)]) (En/2 ((r3(r1+r2+r3)-r1 r2)EllipticK[kr^2]+(r2-r3)(r1+r2+r3+
+  r4)EllipticPi[hr,kr^2]+(r1-r3)(r2-r4)EllipticE[
+  kr^2])+2M En(r3 EllipticK[kr^2]+(r2-r3)EllipticPi[hr,
+  kr^2])+(2M)/(rp-rm) (((4M^2 En-a L)rp-2M a^2 En)/(r3-rp) (EllipticK[
+  kr^2]-(r2-r3)/(r2-rp) EllipticPi[hp,kr^2])-((4M^2 En-
+  a L)rm-2M a^2 En)/(r3-rm) (EllipticK[kr^2]-(r2-r3)/(r2-
+  rm) EllipticPi[hm,kr^2])));*)
+  <|"\!\(\*SubscriptBox[\(\[CapitalUpsilon]\), \(r\)]\)" -> \
+\[CapitalUpsilon]r, 
+   "\!\(\*SubscriptBox[\(\[CapitalUpsilon]\), \(\[Theta]\)]\)" -> 
+    Abs[\[CapitalUpsilon]\[Theta]], 
+   "\!\(\*SubscriptBox[\(\[CapitalUpsilon]\), \(\[Phi]\)]\)" -> \
+\[CapitalUpsilon]\[Phi], 
+   "\!\(\*SubscriptBox[\(\[CapitalUpsilon]\), \(t\)]\)" -> \
+\[CapitalGamma]|>
```

Comment:
Avoid making whitespace changes unless you're doing whitespace cleanup (and if so then try to do that in a separate commit so it's easy to ignore)

Reply:
I wasn't paying attention to the whitespace. I think that just happened from pasting between the raw file and the Mma notebook.